app/src/main/res/layout/fragment_letter_list.xml

```diff
@@ -13,19 +14,17 @@
      See the License for the specific language governing permissions and
      limitations under the License.
 -->
-
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".DetailListFragment">
+    tools:context=".LetterListFragment">
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/recycler_view"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:clipToPadding="false"
-        android:padding="16dp"
-        tools:listitem="@layout/item_view" />
```

Comment:
is this not needed anymore?

Reply:
The fragment's contents are unchanged, and this is to ensure the right view (the teal button that takes up the entire horizontal space) is used for each word.